BREEZE.rst

```diff
@@ -0,0 +1,753 @@
+
+.. code-block:: bash
+
+     ./breeze  --static-check mypy -- --files tests/hooks/test_druid_hook.py
+
+Run all tests on currently staged files:
+
+.. code-block:: bash
+
+     ./breeze  --static-check all
+
+Run all tests on all files:
+
+.. code-block:: bash
+
+     ./breeze  --static-check-all-files all
+
+Run all tests but pylint on all files:
+
+.. code-block:: bash
+
+     ./breeze  --static-check-all-files all-but-pylint
+
+Run pylint checks for all changed files:
+
+.. code-block:: bash
+
+     ./breeze  --static-check pylint
+
+Run pylint checks for selected files:
+
+.. code-block:: bash
+
+     ./breeze  --static-check pylint -- --files airflow/configuration.py
+
+
+Run pylint checks for all files:
+
+.. code-block:: bash
+
+     ./breeze --static-check-all-files pylint
+
+
+The ``license`` check is also run via separate script and separate docker image containing
+Apache RAT verification tool that checks for Apache-compatibility of licences within the codebase.
+It does not take pre-commit parameters as extra args.
+
+.. code-block:: bash
+
+     ./breeze --static-check-all-files licenses
+
+Building the documentation
+--------------------------
+
+The documentation is build using ``-O``, ``--build-docs`` command:
+
+.. code-block:: bash
+
+     ./breeze --build-docs
+
+
+Results of the build can be found in ``docs/_build`` folder. Often errors during documentation generation
+come from the docstrings of auto-api generated classes. During the docs building auto-api generated
+files are stored in ``docs/_api`` folder - so that in case of problems with documentation you can
+find where the problems with documentation originated from.
+
+Running tests
+-------------
+
+If you wish to run tests only and not drop into shell, you can run them by providing
+-t, --test-target flag. You can add extra nosetest flags after -- in the commandline.
+
+.. code-block:: bash
+
+     ./breeze --test-target tests/hooks/test_druid_hook.py -- --logging-level=DEBUG
+
+You can run the whole test suite with special '.' test target:
+
+.. code-block:: bash
+
+    ./breeze --test-target .
+
+You can also specify individual tests or group of tests:
+
+.. code-block:: bash
+
+    ./breeze --test-target tests.core:CoreTest
+
+Running commands inside Docker
+------------------------------
+
+If you wish to run other commands/executables inside of Docker environment you can do it via
+-x, --execute-command flag. Note that if you want to add arguments you should specify them
+together withe the command surrounded with " or ' or pass them after -- as extra arguments.
+
+.. code-block:: bash
+
+     ./breeze --execute-command "ls -la"
+
+.. code-block:: bash
+
+     ./breeze --execute-command ls -- --la
+
+
+Running Docker Compose commands
+-------------------------------
+
+If you wish to run docker-compose command (such as help/pull etc. ) you can do it via
+-d, --docker-compose flag. Note that if you want to add extra arguments you should specify them
```

Comment:
This is just my preference, not mandatory. You can ignore this, I don't have a strong opinion :)

```suggestion
``-d``, ``--docker-compose`` flag. Note that if you want to add extra arguments you should specify them
```